src/table/settings/gui_settings.ini

```diff
@@ -779,7 +779,7 @@ cat      = SC_EXPERT
 var      = gui.ai_developer_tools
 flags    = SF_NOT_IN_SAVE | SF_NO_NETWORK_SYNC
 def      = false
-post_cb  = [](auto) { InvalidateWindowClassesData(WC_SCRIPT_SETTINGS); }
+post_cb  = [](auto) { InvalidateWindowClassesData(WC_SCRIPT_SETTINGS); InvalidateWindowData(WC_GAME_OPTIONS, WN_GAME_OPTIONS_GS); InvalidateWindowClassesData(WC_SCRIPT_DEBUG); }
```

Comment:
Why is this being changed now? Was it missed in #10058?

Reply:
`InvalidateWindowData(WC_GAME_OPTIONS, WN_GAME_OPTIONS_GS)` was missing in #10058
`InvalidateWindowClassesData(WC_SCRIPT_DEBUG)` is added to supposedly show or hide the debug break box when ai_developer_tools setting change, though the invalidate code doesn't really do that yet.

Reply:
I'm going to mark the PR as a draft until you resolve that, just so it's clear that it's not ready for review. Once you fix it, you can mark it ready for review again.